### DISCUSSION TITLE
[BUG] "Not synced or not connected to peers" nearly permanently

### DISCUSSION THREAD

**Discussion by antst** [Chia-Network/chia-blockchain#5036] [Support]
Ubuntu 20.04, LXD container

for last few days farmer nearly always stays in this state.
Sometimes it goes to "Syncing" then "Farming" for a short time, but then drops back to "Not synced or not connected to peers".

There is nothing suspicious in the logs at INFO level.
```chia show -s -c```
shows it is connected to 52 peers

DB is on NVME (which is not used for plotting).
CPU has something like 4 free cores.
1GbE uplink.

**Comment by antst** [comment]
What else can I look at to debug?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
